src/transformers/pipelines/zero_shot_object_detection.py

```diff
@@ -28,11 +20,33 @@
 
 
 @add_end_docstrings(PIPELINE_INIT_ARGS)
-class ZeroShotObjectDetectionPipeline(Pipeline):
+class ZeroShotObjectDetectionPipeline(ChunkPipeline):
     """
     Zero shot object detection pipeline using `OwlViTForObjectDetection`. This pipeline predicts bounding boxes of
     objects when you provide an image and a set of `candidate_labels`.
 
+    Example:
+
+    ```python
+    >>> from transformers import pipeline
+
+    >>> detector = pipeline(model="google/owlvit-base-patch32", task="zero-shot-object-detection")
+    >>> detector(
+    ...     "http://images.cocodataset.org/val2017/000000039769.jpg",
+    ...     candidate_labels=["cat", "couch"],
+    ... )
+    [{'score': 0.287, 'label': 'cat', 'box': {'xmin': 324, 'ymin': 20, 'xmax': 640, 'ymax': 373}}, {'score': 0.254, 'label': 'cat', 'box': {'xmin': 1, 'ymin': 55, 'xmax': 315, 'ymax': 472}}, {'score': 0.121, 'label': 'couch', 'box': {'xmin': 4, 'ymin': 0, 'xmax': 642, 'ymax': 476}}]
+
+    >>> detector(
+    ...         "https://huggingface.co/datasets/Narsil/image_dummy/raw/main/parrots.png",
+    ...         candidate_labels=["head", "bird"],
+    ... )
+    [{'score': 0.119, 'label': 'bird', 'box': {'xmin': 71, 'ymin': 170, 'xmax': 410, 'ymax': 508}}]
```

Comment:
Could we replace this example with a more representative one?

Labels are not actually labels but free text queries ("a photo of a bird", "a close up view of xxx".  It'd also be less confusing to use an example where both queries are found.

Reply:
Then I am confused.

Isn't the pipeline supposed to return part of the image that correspond to the object in question ?
"A photo of  a bird" would be the entire image.
I tried to detect parts of the image other than `bird` but it always failed :(, that's why I used the cat image in the first place.

I do agree that finding an image where we could have different `candidate_labels` for the same image would be a much better test.